mantidproject/mantid
# Update ISIS Logos in Project

**Description of work.**
The logos for ISIS have been out of date since the UKRI rebrand. This replaces them with the new versions. Also replaces the `.pdn` (Paint.NET) file with a `.psd` version, which is supported on more platforms.

Note: The logos on the main webpage aren't in the source so need to be updated separately. Though I'm not sure how to do this myself.

**To test:**
1. Launch workbench, check the splash screen
2. Check any docs that have been updated
3. Check the About Mantidworkbench menu

Fixes #34122 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
